src/domain/platform/admin/space/AdminSpaceListPage/SpaceList.tsx

```diff
@@ -14,13 +15,21 @@ import { useResolvedPath } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
 import { buildAdminSpaceUrl } from '../../../../../main/routing/urlBuilders';
 import SpaceListItem from './SpaceListItem';
+import { sortBy } from 'lodash';
 
 export const SpaceList: FC = () => {
   const { pathname: url } = useResolvedPath('.');
   const notify = useNotification();
   const { t } = useTranslation();
 
   const { data: spacesData, loading: loadingSpaces } = useAdminSpacesListQuery();
+  const { data: organizationData } = useOrganizationsListQuery();
+  const organizations = useMemo(
```

Comment:
I think you're not using `organizations` for anything else, you can save them sorted directly